### PR TITLE
adds judge choice page

### PR DESCRIPTION
**Pull request by BulatCC**

### CODE REVIEW

File: client/src/api/requests/competitions.ts [hunk 1]
```diff
@@ -29,3 +30,23 @@ export const getCompetitionParticipantsApi = async (id: string) => {
 
 	return result
 }
+
+export const getCompetitionJudgesApi = async (id: string) => {
```
Comment (Nurai1) [inline client/src/api/requests/competitions.ts:34]: U need to get all judges from the data base and show them. If competition already has two judges connected, then u can't add judges again.

File: client/src/routes/JudgeChoice.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import { getFormattedDate } from 'src/helpers/datetime'
+import { tableSchemaJudges } from 'src/helpers/tableSchemas/tableSchemaJudges'
+
+const MAX_JUDGES = 2
+
+export const JudgeChoice = (): ReactElement => {
+    const { competitionId } = useParams()
+    const dispatch = useAppDispatch()
+    const [selectedJudges, setSelectedJudges] = useState<string[]>([])
+    const [maxJudgesReached, setMaxJudgesReached] = useState(false)
```
Comment (Nurai1) [inline client/src/routes/JudgeChoice.tsx:17]:
```suggestion
    const maxJudgesReached = selectedJudges.length >= MAX_JUDGES
```

File: server/routes/competitionRoutes.ts [hunk 3]
```diff
@@ -164,7 +164,7 @@ competitionRouter.patch(
             schema: { $ref: '#/definitions/Competition' }
     } */
   UserController.allowIfLoggedin,
-  UserController.grantAccess(ACTIONS.updateAny, RESOURCES.COMPETITION),
```
Comment (Nurai1) [inline server/routes/competitionRoutes.ts:167]: Check your pr before sending it to other developer!

**Comment by BulatCC** [comment]
combined with judge_to_pair_assign_page

**Pull request closed without merging**